libcovebods/config.py

```diff
@@ -10,9 +11,17 @@
 LIB_COVE_BODS_CONFIG_DEFAULT.update({
     'schema_url': os.path.join(_schema_folder, 'schema.json'),
     'schema_url_host': _schema_folder,
+    # These default values are very wide on purpose. It is left to apps using this to tighten them up.
+    'bods_additional_checks_person_birthdate_min_year': 1,
+    'bods_additional_checks_person_birthdate_max_year': datetime.datetime.now().year,
 })
 
 
 class LibCoveBODSConfig(LibCoveConfig):
```

Comment:
I'm not sure about this, compared to just passing a dict around (and remembering to do the appropriate copys).

Reply:
Oh, I see that isn't new to this PR, so approving.

Reply:
Let's talk about the config / copying, would be good to get your opinion.